client/chainlink_models.go

```diff
@@ -402,3 +405,23 @@ func ObservationSourceSpec(url string) string {
 			parse    [type=jsonparse path="data,result"];    
 			fetch -> parse;`, url)
 }
+
+// ObservationSourceConsumerTxEncode spec to put back "directrequest" job data on-chain
+func ObservationSourceConsumerTxEncode(url string) string {
```

Comment:
From looking at this, looks like it should be its own gotemplate object like the others.